### ISSUE TITLE
UI Adjustments

### ISSUE THREAD

**Issue by philsinatra** [hgm27/HM-Website#1]
Here are some adjustments you should consider to enhance your interface:

1. Fix `<nav>` element display and clean up unused styles.
1. Center the images at the bottom of the content area (on both pages).
1. Adjust the hover styles for the links in the footer so they are more obvious.

## `<nav>` Element Adjustments

1. [`position: sticky;` is not implemented properly](https://github.com/hgm27/HM-Website/blob/master/stylesheet.css#L96), and is not doing anything for you. You can safely remove this rule.
1. [`height: 10vh;`](https://github.com/hgm27/HM-Website/blob/master/stylesheet.css#L97) is forcing the height of the container to be larger than needed in some cases. The content within the `<nav>` will automatically adjust the height of the container for you. You can safely remove this rule.
1. [`margin: 0;`](https://github.com/hgm27/HM-Website/blob/master/stylesheet.css#L98) isn't doing anything for you and can safely be removed.

## Center Images

If you wrap the images at the bottom of both screens in a `<div>` and use CSS to center the content within, the images will be centered instead of to the left. Simplest way is probably to use `text-align: center;`

## Footer Hover Styles

The hover color changes for the links in the footer are not drastic enough. Consider someone who is color blind - would they notice the change in color? Try something with more contrast so it's obvious.

![himamistry](https://user-images.githubusercontent.com/1465808/33941892-2c177c3a-dfe2-11e7-996c-a045259421fd.png)

**Comment by hgm27** [comment]
Are you suggesting to move the images out of the div I already have into a new one?

**Comment by philsinatra** [comment]
No, I'm suggesting adding an additional `<div>` just for the three images, then using CSS to center the content (images) within the `<div>`.

**Comment by philsinatra** [comment]
Adjustments look good. I would make one more small change and I think you're good to go. Move your new `pictures` container down a bit for more consistent vertical spacing. Probably easiest to use`margin` or `padding` to accomplish this.

![himamistry](https://user-images.githubusercontent.com/1465808/33995982-a911cada-e0ad-11e7-861a-3e28448e195f.png)
